eclair-core/src/test/resources/integration/bitcoin.conf

```diff
@@ -1,11 +1,13 @@
 regtest=1
+noprinttoconsole=1 
 server=1
 port=28333
-rpcport=28332
+regtest.rpcport=28332
 rpcuser=foo
 rpcpassword=bar
 txindex=1
 zmqpubrawblock=tcp://127.0.0.1:28334
 zmqpubrawtx=tcp://127.0.0.1:28335
 rpcworkqueue=64
 addresstype=bech32
+deprecatedrpc=signrawtransaction
```

Comment:
We should instead use the new method for signing transactions, and tell users to update their bitcoin core. Testing version 0.17.3 makes it the currently supported version, so we need to be consistent and be up-to-date with this version.